lib/qs.rb

```diff
@@ -48,6 +61,9 @@ def self.redis_config
   class Config
     include NsOptions::Proxy
 
+    option :serializer,   Proc, :default => proc{ |p| ::JSON.dump(p) }
+    option :deserializer, Proc, :default => proc{ |p| ::JSON.load(p) }
```

Comment:
@kellyredding - I added a commit wiht tweaks to this, the real issue with requiring and a diff var name are line length and ugliness, you get something like:

```ruby
option :deserializer, Proc, :default => proc{ |serialized_payload| require 'json'; ::JSON.load(serialized_payload) }
# OR multi-lining the proc:
option :deserializer, Proc, :default => proc do |serialized_payload|
  require 'json'
  ::JSON.load(serialized_payload)
end
```

I find the multi-line harder to read, but it's ok. On the single line the `JSON.load` is passed 80 chars so you can't see the meat of the proc without possibly scrolling/wrapping.

I like the features but we pay a style price. The other issue is I got a much worse benchmark for writing the jobs with this change. I'm gonna try restarting and do it again, see if my machine is just taxed for some reason. I'll let you know.